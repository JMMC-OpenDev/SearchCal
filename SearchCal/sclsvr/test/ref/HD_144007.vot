version https://git-lfs.github.com/spec/v1
oid sha256:71616852d654a063f3c92820139d91330cb03d01e7599b363f5679847e4e72d3
size 129203
