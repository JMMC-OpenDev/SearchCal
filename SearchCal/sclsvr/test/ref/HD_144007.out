version https://git-lfs.github.com/spec/v1
oid sha256:a7a882cd82be371ade0eb240a482f53a013a2e015b91530668118dcabfbbf66e
size 161097
